src/EditorFeatures/TestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -76,17 +78,25 @@ public Test()
             /// </summary>
             public LanguageVersion LanguageVersion { get; set; } = LanguageVersion.CSharp8;
 
+#if !CODE_STYLE // TODO: Add support for Options based tests in CodeStyle layer
             /// <summary>
             /// Gets a collection of options to apply to <see cref="Solution.Options"/> for testing. Values may be added
             /// using a collection initializer.
             /// </summary>
             public OptionsCollection Options { get; } = new OptionsCollection(LanguageNames.CSharp);
+#endif
 
             public Func<ImmutableArray<Diagnostic>, Diagnostic?>? DiagnosticSelector { get; set; }
 
             protected override AnalyzerOptions GetAnalyzerOptions(Project project)
             {
-                return new WorkspaceAnalyzerOptions(base.GetAnalyzerOptions(project), project.Solution);
+                var analyzerOptions = base.GetAnalyzerOptions(project);
+
+#if !CODE_STYLE
+                analyzerOptions = new WorkspaceAnalyzerOptions(analyzerOptions, project.Solution);
+#endif
+
+                return analyzerOptions;
             }
```

Comment:
💡 Just wrap the entire method. The inherited implementation works for the code style layer.